api/tests/unit/edge_api/identities/test_mixins.py

```diff
@@ -0,0 +1,273 @@
+import json
+
+from edge_api.identities.mixins import (
+    MigrateIdentitiesUsingRequestsMixin,
+    MigrateTraitsUsingRequestMixin,
+    _should_migrate,
```

Comment:
We're testing a lot of 'private' methods and functions in this module. Is there a way to refactor so we're not doing that? 

Some options: 

 1. Could we avoid testing `_should_migrate` entirely and just test the methods that call it for cases when it should / shouldn't migrate the data
 2. Should the migrate_sync and migrate_async functions both be 'public'? 
 3. Is there a better way to architect this? It does feel a _little_ bit hacky that we have to, for example, set the default on the setting to string because we're _always_ initialising the mixin. I wonder if this shouldn't be a mixin?

Reply:
Yes, I do feel mixin was not the best choice for this. I decided to move towards a bit more functional approach to solve this, and the code is certainly a bit cleaner now.

Reply:
Yep, I definitely agree. I've added a few smaller comments to specific areas I think can be improved but overall, I definitely prefer this approach.